pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/columnminmaxvalue/ColumnMinMaxValueGenerator.java

```diff
@@ -56,32 +58,16 @@ public void addColumnMinMaxValue()
     Preconditions.checkState(_columnMinMaxValueGeneratorMode != ColumnMinMaxValueGeneratorMode.NONE);
 
     Schema schema = _segmentMetadata.getSchema();
+    Set<String> columnsToAddMinMaxValue = new HashSet<>(schema.getPhysicalColumnNames());
 
-    // Process time column
-    String timeColumnName = schema.getTimeColumnName();
-    if (timeColumnName != null) {
-      addColumnMinMaxValueForColumn(timeColumnName);
+    switch (_columnMinMaxValueGeneratorMode) {
+      case TIME:
+        columnsToAddMinMaxValue.removeAll(schema.getDimensionNames());
```

Comment:
can u add a comment here that we intentionally fall through to next? thanks

Reply:
Done